src/Tizen.NUI.Scene3D/src/internal/Interop/Interop.MotionData.cs

```diff
@@ -0,0 +1,87 @@
+﻿/*
+ * Copyright(c) 2023 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+namespace Tizen.NUI.Scene3D
+{
+    internal static partial class Interop
+    {
+        internal static partial class MotionData
+        {
+            [global::System.Runtime.InteropServices.DllImport(Libraries.Scene3D, EntryPoint = "CSharp_Dali_MotionData_New_SWIG_0")]
```

Comment:
very great patch! two thumbs up!
just minor comment for next modification.

https://github.com/Samsung/TizenFX/blob/master/src/Tizen.NUI/src/internal/Interop/Interop.Accessibility.cs
like above, by `using global::System;  using System.Runtime.InteropServices;` we can make it simpler. ^^;

you can fix it from next time.~